nuxeo-opencmis-tests/src/test/resources/OSGI-INF/elasticsearch-test-contrib.xml

```diff
@@ -1,45 +1,5 @@
 <?xml version="1.0"?>
-<component name="org.nuxeo.elasticsearch.test.contrib">
-
-  <extension target="org.nuxeo.elasticsearch.ElasticSearchComponent" point="elasticSearchLocal">
-    <elasticSearchLocal clusterName="nuxeoTestClusterLocal" nodeName="nuxeoTestNode"
-      indexStoreType="mmapfs" httpEnabled="true" />
-  </extension>
-
-  <extension target="org.nuxeo.elasticsearch.ElasticSearchComponent" point="elasticSearchIndex">
-    <elasticSearchIndex name="nxutest" type="doc" repository="test" />
-    <elasticSearchIndex name="nuxeo-uidgen" type="seqId">
-      <settings>
-        {
-        "number_of_shards" : 1,
-        "auto_expand_replicas" : "0-all"
-        }
-      </settings>
-      <mapping>
-        {
-        "_source" : {"enabled": false},
-        "_all" : {"enabled": false},
-        "enabled" : false
-        }
-      </mapping>
-    </elasticSearchIndex>
-    <elasticSearchIndex name="nuxeo-audit" type="entry" >
-      <mapping>
-        {
-          "properties" : {
-            "eventDate": {
-              "format": "dateOptionalTime",
-              "type": "date"
-            },
-            "logDate": {
-              "format": "dateOptionalTime",
-              "type": "date"
-            }
-          }
-        }
-      </mapping>
-    </elasticSearchIndex>
-  </extension>
+<component name="org.nuxeo.elasticsearch.test.cmis.contrib">
 
   <!-- for uidgen which is a dependency -->
   <extension target="org.nuxeo.runtime.datasource" point="datasources">
```

Comment:
Couldn't we rely in the tests on `org.nuxeo.elasticsearch.audit.test:nxuidsequencer-ds.xml` (which might need to be parametrized) and get rid of nuxeo-opencmis-tests/src/test/resources/OSGI-INF/elasticsearch-test-contrib.xml?

Reply:
yes it would be better but it was not working, so for the next time